fatips/300.md

```diff
@@ -0,0 +1,438 @@
+          "extIds":[
+           "fb736f345dd5d67ac26eab94c9c9f51b708b05c",
+           "qifu3ino28381n2937n1293mi912kfmlasfkasdkmfasdfasdf",
+           "f2on3p2u123n712n387nf19msov2jbmurbhimsdo",
+           "mvnansuvbeybqyyyewvyqv273212oi3onbvttbywtybhxzzxuv",
+       ]
+      },
+      {  
+         "txId":"80d87a8bd5cf2a3eca9037c2229f3701eed29360caa975531ef5fe476b1b70b5",
+         "inputs:":[  
+            {  
+               "address":"FA1zT4aFpEvcnPqPCigB3fvGu4Q4mTXY22iiuV69DqE1pNhdF2MC",
+               "amount":100
+            },
+            {  
+               "address":"FA2y6VYYPR9Y9Vyy1ZuZqWWRXGXLeuvsLWGkDxq3Ed7yc11dbBKV",
+               "amount":50
+            }
+         ],
+         "outputs:":[  
+            {  
+               "address":"FA3aECpw3gEZ7CMQvRNxEtKBGKAos3922oqYLcHQ9NqXHudC6YBM",
+               "amount":150
+            }
+         ],
+         "milliTimestamp":153745188,
+         "salt":"i2n3f8923n98f2981sd701eed29360234234n2jnb5",
+          "extIds":[
+           "fb736f345dd5d67ac26eab94c9c9f51b708b05c",
+           "qifu3ino28381n2937n1293mi912kfmlasfkasdkmfasdfasdf",
+           "f2on3p2u123n712n387nf19msov2jbmurbhimsdo",
+           "mvnansuvbeybqyyyewvyqv273212oi3onbvttbywtybhxzzxuv",
+       ]
+      }
+   ]
+}
+```
+
+(note: extIds shown are not realistic values)
+
+<br/>
+
+### `get-balance` :
+
+Get the balance of an address for a token
+
+#### Parameters:
+
+| Name         | Type   | Description                               | Validation                      | Required |
+| ------------ | ------ | ----------------------------------------- | ------------------------------- | -------- |
+| `fa-address` | string | The Factoid address to get the balance of | Must be a valid Factoid address | Y        |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+   "result": 2321111
+}
+```
+
+<br/>
+
+### `get-stats` :
+
+Get overall statistics for a token
+
+#### Parameters:
+
+| Name | Type | Description | Validation | Required |
+| ---- | ---- | ----------- | ---------- | -------- |
+|      |      |             |            |          |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+    "result":{
+        "supply":10000000,
+        "circulatingSupply":8113,
+        "transactions":18429,
+        "issuanceTimestamp": 1518286500,
+        "lastTransactionTimestamp": 1539212767
+    }
+}
+```
+
+
+
+### `get-nf-token` :
+
+Get a non fungible token by ID. The token belong to non fungible token class.
+
+#### Parameters:
+
+| Name          | Type   | Description                             | Validation                                 | Required |
+| ------------- | ------ | --------------------------------------- | ------------------------------------------ | -------- |
+| `nf-token-id` | string | The unique ID of the non fungible token | Must be a valid issued non fungible token. | Y        |
```

Comment:
there is no need for the chain/issuer here? The tokens ID are not going to be globally unique, are they?

Reply:
The issuer root chain ID + tokenID or chain ID are required for all token methods (see the top section of token methods header)

Reply:
Hm ok that's a bit disconcerting. I guess if we consider it is a spec and not a doc it may be fine. But as a developer I'd certainly jump right to the API I want to learn about without checking all the preamble.

Reply:
Thanks for the input, we can always add it back later. It's a bit more maintainable to have it it one place for now until we finalize things :+1: